internal/linter/jsonschema/jsonschema_lint.go

```diff
@@ -0,0 +1,230 @@
+package linter
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+// JSONSchemaLinter checks for unescaped commas in jsonschema struct tags
+type JSONSchemaLinter struct {
+	FilePaths []string
+	Errors    []JSONSchemaError
+	FixMode   bool
+	Fixed     map[string]bool
+}
+
+// JSONSchemaError represents a linting error with file position details
+type JSONSchemaError struct {
+	FilePath  string
+	Line      int
+	Column    int
+	Offset    int   // Byte offset in the file
+	Struct    string
+	Field     string
+	Tag       string
+	FixedTag  string
+}
+
+// tagPattern matches jsonschema tags with description containing unescaped commas
+// It captures:
+// 1. The jsonschema tag
+// 2. Parts of the description containing unescaped commas
+var tagPattern = regexp.MustCompile(`jsonschema:"([^"]*)description=([^"]*[^\\],[^"]*)"`) 
```

Comment:
Does this fail if we use escaped quotes then a comma inside of a jsonschema tag?